app/react/Documents/components/specs/DocumentSidePanel.spec.js

```diff
@@ -108,6 +109,27 @@ describe('DocumentSidePanel', () => {
         ).toBe('metadata');
       });
     });
+
+    describe('when doc passed has toc', () => {
+      const toc = ['title1', 'title2', 'title3'];
+
+      it('if is a new entity should set toc of the loaded file', () => {
+        props.doc = Immutable.fromJS({ metadata: [], attachments: [], type: 'entity' });
+        props.file = { toc, pdfInfo: {} };
+        props.tab = 'toc';
+        render();
+        expect(component.find({ toc })).toHaveLength(1);
+      });
+
+      it('if is a loaded entity should set the toc of the default document', () => {
```

Comment:
Please, apply the same principle as before: either nest the "if" in a new describe, or start with "should" and move the conditional to the end of the pharse.